sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/DecryptionResult.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+package com.azure.cosmos.encryption;
+
+public class DecryptionResult {
```

Comment:
You should probably start with all these classes as `final`

Reply:
will do. thanks.

Reply:
addressed.